service/routes.go

```diff
@@ -93,7 +93,8 @@ func driverSchemas() *v1client.Schemas {
 			field.Type = key
 			field.Create = true
 			webhook.ResourceFields[webhookField] = field
-			driverConfig := schemas.AddType(key, value.GetSchema())
+			driverConfig := schemas.AddType(key, value.GetDriverConfigResource())
+			driverConfig = value.CustomizeSchema(driverConfig)
```

Comment:
Move this below the loop that sets all the fields to create so that driver changes dont get stomped on by the framework